executor/src/lib.rs

```diff
@@ -416,6 +417,7 @@ pub async fn batch_process(
 
     let output_path = format!("{}/{}/{}", base_dir, task_id, task);
     log::info!("output_path: {}", output_path);
+    std::fs::create_dir_all(output_path.clone()).expect("Failed to create directory");
```

Comment:
Don't use expect here, use ` .unwrap_or_else(|_| panic!("Failed to write to file, slot_path: {}", slot_path))` to print the path   when panic